test/e2e/metamask-ui.spec.js

```diff
@@ -772,7 +772,7 @@ describe('MetaMask', function () {
       await driver.delay(regularDelayMs * 2)
 
       await driver.clickElement(By.xpath(`//button[contains(text(), 'Create Token')]`))
-      await waitUntilXWindowHandles(3)
+      await driver.waitUntilXWindowHandles(3)
```

Comment:
Oh, huh, I think the handles need to be re-fetched after this line. Or maybe `waitUntilXWindowHandles` could return them directly.

The list of handles retrieved above likely is stale.

Reply:
Good catch: 87c9e79